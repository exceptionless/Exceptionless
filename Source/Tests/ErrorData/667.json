{
  "Id": "80000000b77efe3144d266fc",
  "OccurrenceDate": "2013-03-27T09:51:41.2857762-05:00",
  "Tags": [],
  "ExceptionlessClientInfo": {
    "Version": "1.0.0.667",
    "InstallDate": "0001-01-01T00:00:00",
    "StartCount": 0,
    "SubmitCount": 0,
    "Platform": ".NET"
  },
  "Modules": [],
  "EnvironmentInfo": {
    "ProcessorCount": 12,
    "TotalPhysicalMemory": 17120079872,
    "AvailablePhysicalMemory": 10080608256,
    "CommandLine": "\"C:\\Users\\Example\\Documents\\Visual Studio 2012\\Projects\\MvcApplication5\\ConsoleApplication1\\bin\\Debug\\ConsoleApplication1.vshost.exe\" ",
    "ProcessName": "C:\\Users\\Example\\Documents\\Visual Studio 2012\\Projects\\MvcApplication5\\ConsoleApplication1\\bin\\Debug\\ConsoleApplication1.vshost.exe",
    "ProcessId": "12612",
    "ProcessMemorySize": 19161088,
    "ThreadId": "11572",
    "Architecture": "x64",
    "OSName": "Microsoft Windows 8 Enterprise",
    "OSVersion": "6.2.9200.0",
    "MachineName": "Example",
    "RuntimeVersion": "4.0.30319.18033",
    "IpAddress": "127.0.0.1",
    "ExtendedData": {
      "AppDomainName": "ConsoleApplication1.vshost.exe"
    }
  },
  "Message": "Exception of type 'System.Exception' was thrown.",
  "Type": "System.Exception",
  "ExtendedData": {
    "ExtraExceptionProperties": "{\r\n  \"Data\": {}\r\n}"
  },
  "StackTrace": []
}