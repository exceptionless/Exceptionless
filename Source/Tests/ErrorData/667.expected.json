{
  "ReferenceId": "80000000b77efe3144d266fc",
  "Date": "2013-03-27T09:51:41.2857762-05:00",
  "@environment": {
    "ProcessorCount": 12,
    "TotalPhysicalMemory": 17120079872,
    "AvailablePhysicalMemory": 10080608256,
    "CommandLine": "\"C:\\Users\\Example\\Documents\\Visual Studio 2012\\Projects\\MvcApplication5\\ConsoleApplication1\\bin\\Debug\\ConsoleApplication1.vshost.exe\" ",
    "ProcessName": "C:\\Users\\Example\\Documents\\Visual Studio 2012\\Projects\\MvcApplication5\\ConsoleApplication1\\bin\\Debug\\ConsoleApplication1.vshost.exe",
    "ProcessId": "12612",
    "ProcessMemorySize": 19161088,
    "ThreadId": "11572",
    "Architecture": "x64",
    "OSName": "Microsoft Windows 8 Enterprise",
    "OSVersion": "6.2.9200.0",
    "MachineName": "Example",
    "RuntimeVersion": "4.0.30319.18033",
    "IpAddress": "127.0.0.1",
    "Data": {
      "AppDomainName": "ConsoleApplication1.vshost.exe"
    }
  },
  "Message": "Exception of type 'System.Exception' was thrown.",
  "Type": "error",
  "@error": {
    "Message": "Exception of type 'System.Exception' was thrown.",
    "Type": "System.Exception"
  }
}